### DISCUSSION TITLE
Cannot specify icon in spreadsheet that doesn't have a direct URL

### DISCUSSION THREAD

**Discussion by Wallby** [Devolutions/UniGetUI#1847] [Q&A]
For the VulkanSDK there is an icon which can be downloaded here.. https://www.khronos.org/legal/trademarks/

The specific link is https://members.khronos.org/document/dl/4781.
This downloads a file vulkan.zip in which the icon is in `Vulkan(r)_Nov17\Vulkan for web\Vulkan Logo Bug\Vulkan_LogoBug_48px_Nov17.png`.

**Comment by Wallby** [comment]
The image on chocolatey..
https://community.chocolatey.org/packages/vulkan-sdk
.. violates the logo guidelines as specified here.. https://www.khronos.org/files/legal/Khronos-Logo-Usage-Guide.pdf

**Comment by marticliment** [comment]
But Vulkan does not have an associated icon on WingetUI

![image](https://github.com/marticliment/WingetUI/assets/53119851/06445c3f-9e6c-455d-b0c1-1dc69746a70a)

**Reply by Wallby** [reply]
>  The image on chocolatey..
https://community.chocolatey.org/packages/vulkan-sdk
.. violates the logo guidelines as specified here.. https://www.khronos.org/files/legal/Khronos-Logo-Usage-Guide.pdf

I said this because there are other URLs in the spreadsheet that use chocolatey for the icon, and though there is an icon for the vulka sdk package on chocolatey, that should not be used as it is wrong.

**Reply by Wallby** [reply]
This icon violates the vulkan logo guidelines, which are very explicit.. https://community.chocolatey.org/content/packageimages/vulkan-sdk.1.2.182.0.png

**Reply by marticliment** [reply]
Then you may want to report this to the respective package maintainers

**Reply by Wallby** [reply]
I mentioned it only as a heads up to not use that icon here. But this discussion is about the question can an icon be used without an URL to that icon? Because there is an icon that could be used for winget Vulkan SDK but no URL to it.

**Reply by marticliment** [reply]
You can upload the icon to some service such as imgur
